### PR TITLE
Fixed home view and added static keyword to login and register view

### PR DESCRIPTION
**Pull request by ikennaumeh**

### CODE REVIEW

File: lib/ui/views/homepage/widgets/product_item.dart [hunk 1]
```diff
@@ -33,12 +31,11 @@ class ProductItem extends StatelessWidget {
             Stack(
               alignment: AlignmentDirectional.topEnd,
               children: [
-                Image.asset(
-                  recipe?.dishImageUrl ?? '--',
+                Image.network(
+                  recipe?.dishImageUrl ?? 'https://media.istockphoto.com/id/1404501005/photo/jollof-rice.jpg?s=1024x1024&w=is&k=20&c=DTns3r_STc4HRBMouVl8IgHeuWhnBdz8X4DRkbDxWjQ=',
```
Comment (moak13) [inline lib/ui/views/homepage/widgets/product_item.dart:35]: This is actually not the solution to our issue. I told them in the last class what needs to be done and someone should create a PR for it. Men, it's well.

File: lib/ui/views/login/login_view.dart [hunk 2]
```diff
@@ -21,7 +21,7 @@ import 'login_viewmodel.dart';
 class LoginView extends StackedView<LoginViewModel> with $LoginView {
   LoginView({Key? key}) : super(key: key);
 
-  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  static final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
```
Comment (moak13) [inline lib/ui/views/login/login_view.dart:24]: This is going to be a routing blow-up, always check the things you add for a temporary fix.

File: lib/ui/views/signup/signup_view.dart [hunk 3]
```diff
@@ -24,7 +24,7 @@ import 'signup_viewmodel.dart';
 class SignupView extends StackedView<SignupViewModel> with $SignupView {
   SignupView({Key? key}) : super(key: key);
 
-  final GlobalKey<FormState> _signupKey = GlobalKey<FormState>();
+  static final GlobalKey<FormState> _signupKey = GlobalKey<FormState>();
```
Comment (moak13) [inline lib/ui/views/signup/signup_view.dart:27]: same as above

File: lib/data_model/register_model.dart [hunk 4]
```diff
@@ -4,7 +4,7 @@ class RegisterModel {
 
   Map<String, dynamic> toJSON() {
     return {
-      "id": 0,
+      "id": 1,
```
Comment (moak13) [inline lib/data_model/register_model.dart:7]: No need to pass the id when registering as well. Was this causing an issue?

Comment (ikennaumeh) [reply lib/data_model/register_model.dart:7]: Oh for this, the registration wasn't working so I tried to fix things but that didn't work still

File: lib/ui/views/login/login_view.dart (hunk 2 above)
Comment (ikennaumeh) [reply lib/ui/views/login/login_view.dart:24]: That's the thing, nothing blows up from my end. And that's the only fix for keyboard disappearing as soon as it's called.

https://github.com/flutter/flutter/issues/20042

File: lib/ui/views/homepage/widgets/product_item.dart (hunk 1 above)
Comment (ikennaumeh) [reply lib/ui/views/homepage/widgets/product_item.dart:35]: Ohh I missed this. I no no wetin u tell them. I just saw the null error and made it work

**Pull request merged**